app/scripts/modules/core/src/presentation/forms/inputs/JsonEditor.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @spinnaker/import-sort */
```

Comment:
Explicitly turning off to ensure existing import order that relies on side effects.